.github/workflows/buildpack-integration-test.yml

```diff
@@ -2,51 +2,54 @@
 name: Buildpack Integration Test
 on:
   push:
-    branches:
-      - master
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
   workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
Can you add a comment here on how often this runs?

Reply:
Nice catch, Done.